### PR TITLE
Add pacemaker(pacemaker-based) resource translatefile

### PR DESCRIPTION
**Pull request by daaitudian**
Hi,
@kgaillot ,Thanks for your work. I resubmit this PR based on the current branch. 

Regarding the new proposal, I agree with you, when we have more people to support us to maintain the translation in the future, we can consider enable it by default again .

### CODE REVIEW

**Comment by wenningerk** [comment]
test this please

**Comment by daaitudian** [comment]
Hi,
In the latest commit I set the translations into disabled by default , user can use translations by  "./configure --enable-nls"

**Comment by kgaillot** [comment]
retest this please

(for CI bot)

**Review by gao-yan — COMMENTED** [review]
Thanks for the good idea and work. Just a few nitpickings.

**Comment by kgaillot** [comment]
Hi @daaitudian , let me know if you want to respond to @gao-yan 's review, and I'm ready to merge

File: po/zh_CN.po [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+"设置为0将禁用轮询，设置为正数将是以秒为单位的时间间隔（除非使用了其他单位，比如\"5min\"表示5分钟）"
+
+#: lib/pengine/common.c:132
+msgid "Whether watchdog integration is enabled"
+msgstr "是否启用看门狗集成设置"
+
+#: tools/crm_resource.c:1418
+#, c-format
+msgid "'%s' is not a valid agent specification"
+msgstr "'%s' 是一个无效的代理"
```
[the remark this answers is not in the document]
Comment (daaitudian) [reply po/zh_CN.po:53]: Thank you for your suggestions, which are very helpful for my translation. I will also pay attention to these issues in future translations.

**Comment by daaitudian** [comment]
Thank you for reminding me. I have revised the translation problem. ：）

**Pull request merged**